hasher-matcher-actioner/webapp/src/pages/Dashboard.jsx

```diff
@@ -126,13 +127,23 @@ function StatCard({statName, timeSpan}) {
     <Card key={`stat-card-${statName}`} className="mb-4">
       <Card.Header>
         <GraphWithNumberWidget graphData={toUFlotFormat(card.graph_data)} />
-        <h1>{getDisplayNumber(card.number)}</h1>
       </Card.Header>
       <Card.Body>
-        <Card.Title>{getDisplayTitle(statName)}</Card.Title>
-        <Card.Subtitle>
-          Processed in the last {getDisplayTimeSpan(card.time_span)}.
-        </Card.Subtitle>
+        <Row>
+          <Col xs={4}>
+            <h3 style={{fontWeight: 300}}>{getDisplayTitle(statName)}</h3>
+          </Col>
+          <Col xs={4}>
+            <h1>{getDisplayNumber(card.time_span_count)}</h1>
+            <small className="text-muted">
+              in the last {getDisplayTimeSpan(card.time_span)}.
+            </small>
+          </Col>
+          <Col xs={4}>
+            <h1>{getDisplayNumber(card.total_count)}</h1>
+            <small className="text-muted">since HMA is online.</small>
```

Comment:
nit/not blocking: awkward wording (and this value maybe doesn't need to be a large as the count for the time range.) 

something like: "of <total_count> processed in total." somewhere in the card is better imo

Reply:
I see what you mean. I'm not sure either. Would defer to more experienced hands on copy writing. Happy to revisit when we are doing a design/copy thingy.